.golangci.yaml

```diff
@@ -31,3 +32,5 @@ linters-settings:
   funlen:
     lines: 96
     statements: 48
+  lll:
+    line-length: 100
```

Comment:
This disagrees with the line length set in the revive config.  On top of that, is it not redundant if revive is already checking line length?

Reply:
Fixed to lll.line-length=98 (same as revive). Apparently, this linter overlaps checks done by revive.

Reply:
Awesome, thanks!